propolis/src/hw/nvme/cmds.rs

```diff
@@ -607,24 +607,17 @@ impl PrpIter<'_> {
                     // entry which should be present in PRP2
                     PrpNext::Prp2
                 } else {
-                    /*
-                     * The first PRP List entry (i.e. the first pointer to a memory page containing
-                     * additional PRP entries) that if present is contained in the PRP Entry 2 location
-                     * within the command, shall be Qword aligned and may also have a non-zero offset within
-                     * the memory page.
-                     */
+                    // The first PRP List entry (i.e. the first pointer to a memory page containing
+                    // additional PRP entries) that if present is contained in the PRP Entry 2
+                    // location within the command, shall be Qword aligned and may also have a
+                    // non-zero offset within the memory page.
```

Comment:
The phrasing on this is a bit awkward.  It seems like it needs at least a comma, and perhaps be split up.